src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int firstBase;
+    private int secondBase;
+    private int heigh;
+
+    public IsoscelesTrapezoid(int firstBase, int secondBase, int heigh, Color color) {
+        super(color);
+        this.firstBase = firstBase;
+        this.secondBase = secondBase;
+        this.heigh = heigh;
+    }
+
+    @Override
+    public double getArea() {
+        return ((firstBase + secondBase) * heigh) / 2.0;
+    }
+
+    @Override
+    public String getFigureInfo() {
+        return "Figure: IsoscelesTrapezoid, area: " + getArea() + " sq. units, firstBase: "
+                + firstBase + " secondBase "
+                + secondBase + " heigh " + heigh + " units, color: " + getColor();
+    }
+    /* Wzór na pole trapezu równoramiennego (a+b)*h /2 */

```

Comment:
The comment on line 26 is in Polish. It should be translated to English to maintain consistency and ensure that all developers can understand it.